src/pal/src/CMakeLists.txt

```diff
@@ -16,8 +16,37 @@ if(NOT CLR_CMAKE_USE_SYSTEM_LIBUNWIND)
   include_directories(${CMAKE_CURRENT_BINARY_DIR}/libunwind/include/tdep)
 
   add_subdirectory(libunwind)
-endif(NOT CLR_CMAKE_USE_SYSTEM_LIBUNWIND)
+else()
+    if(PAL_CMAKE_PLATFORM_ARCH_ARM)
```

Comment:
This should not be enabled for OSX. I think it might be cause for the OSX test failure.

Reply:
I can add some more if. But it seems like OSX does not have arch specific library so UNWIND_ARCH should end up empty and not be used. I can verify that but if it works that would would that be sufficient?

Reply:
We weren't adding any unwind library for OSX before.